packages/aragon-wrapper/src/identity/LocalIdentityProvider.test.js

```diff
@@ -138,8 +138,8 @@ test.serial('search should return an array of results of freely matching identit
   ]
   // map of search terms to expected count and names
   const searchTermToExpectation = {
-    'ne': { names: [] },
-    '0x': { names: [] },
+    'xn': { names: [] },
```

Comment:
I'd actually add one test for searching with two chars with results, to indicate we're not ruling that out on the provider level

Reply:
Done, added more tests on tests with 1 and 2 chars.